src/components/ListUserSearch/ListUserSearch.js

```diff
@@ -1,14 +1,22 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import {Select} from 'antd';
-import {Avatar} from '../../Avatar';
+import Avatar from '../../Avatar';
 
 import 'antd/dist/antd.css';
 import styles from './ListUserSearch.less'
 
-export function ListUserSearch(props) {
-    const {onSelectUser, usersInfo, owners} = props;
+export default function ListUserSearch(props) {
+    const {onSelectUser, usersData, owners} = props;
     const {Option} = Select;
 
+    let usersOptionSelect = usersData.filter((userData) => (
+        !owners.find(ownerId => userData._id === ownerId)
+    ));
+
+    const headerOnBlur = useCallback(() => {
+        onSelectUser(null);
+    })
```

Comment:
please don't forget about dependencies for `useCallback`

Reply:
const headerOnBlur = useCallback(() => {
        onSelectUser(null);
    }, [onSelectUser])

correct?

Reply:
right